README.md

```diff
@@ -135,6 +135,20 @@ end
 Configuration
 -------------
 
+### Identifiers
```

Comment:
It's my own fault for not skimming through and not reading the config. That's what I was trying to get at, just a little blurb to point to it.